src/main/kotlin/com/criptext/mail/scenes/settings/cloudbackup/CloudBackupScene.kt

```diff
@@ -34,8 +36,8 @@ interface CloudBackupScene{
     fun disableSaveButtonOnDialog()
     fun enableSaveButtonOnDialog()
     fun getGoogleDriveService(): Drive?
-    fun scheduleCloudBackupJob(period: Int, accountId: Long)
-    fun removeFromScheduleCloudBackupJob(accountId: Long)
+    fun scheduleCloudBackupJob(period: Int, recipientId: String)
```

Comment:
`recipientId ` can be daniel and daniel@otherdomain.com ? if don't, this could cause an issue